### ISSUE TITLE
Rename project

### ISSUE THREAD

**Issue by mnafshin** [fallenprogrammr/addtogitignore#4]
After adding some files to .gitignore, I have changed the a sub-module-project's name and also its folder name respectively, but the rename is not synchronized in .gitignore file. (There exist only one .gitignore file in the parent module not in submodules)

**Comment by fallenprogrammr** [comment]
You are correct, there is no syncing mechanism and would be an interesting idea to pursue, the plugin currently triggers from the interaction that an user has with the project tool window in IntelliJ. 

I would have to look into the feasibility of maintaining a background process which maintains a list of gitgnore files and directories and syncs any changes. This would pose a problem if the rename was intended to not be included in .gitignore.
